source/utilities.tex

```diff
@@ -568,6 +572,20 @@
 \tcode{bool}.
 \end{note}
 
+\rSec2[utility.underlying]{Function template \tcode{to_underlying}}
+
+\begin{itemdecl}
+template<class T>
+  constexpr underlying_type_t<T> to_underlying(T value) noexcept;
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\returns
+\tcode{static_cast<underlying_type_t<T>>(value)}.
+\end{itemdescr}
+
+
```

Comment:
```suggestion
```

Reply:
Several other subclauses in Clause 20 (and elsewhere) are double-spaced before a new `rSec1` section.